kubel.el

```diff
@@ -786,6 +820,13 @@ RESET is to be called if the search is nil after the first attempt."
 
 ;; popups
 
+(define-transient-command kubel-exec-popup ()
+  "Kubel Exec Menu"
+  ["Actions"
+   ("f" "Find files" kubel-exec-pod)
```

Comment:
I think this should be `("d" "Dired" kubel-exec-pod)` or something. "Find files" makes me think of `C-x C-f` which is not exactly the same here.

Reply:
Ya, I updated